src/main/java/taxi/model/Driver.java

```diff
@@ -15,6 +17,13 @@ public Driver(String name, String licenseNumber) {
         this.licenseNumber = licenseNumber;
     }
 
+    public Driver(String name, String licenseNumber, String login, String password) {
+        this.name = name;
+        this.licenseNumber = licenseNumber;
+        this.login = login;
+        this.password = password;
+    }
```

Comment:
as described in common mistakes, consider using this(name, licenseNumber) instead of duplicating your code inside constructor